client/src/main/kotlin/no/nav/su/se/bakover/client/krr/KontaktOgReservasjonsregisterClient.kt

```diff
@@ -25,16 +27,31 @@ class KontaktOgReservasjonsregisterClient(
     private val log = LoggerFactory.getLogger(this::class.java)
 
     override fun hentKontaktinformasjon(fnr: Fnr): Either<KontaktOgReservasjonsregister.KunneIkkeHenteKontaktinformasjon, Kontaktinformasjon> {
-        val (_, response, result) = "${config.url}$PERSON_PATH".httpGet()
+        val request = serialize(
+            HentKontaktinformasjonRequest(
+                personidenter = listOf(fnr.toString()),
+            ),
+        )
+        val (_, response, result) = "${config.url}$PERSONER_PATH".httpPost()
             .authentication().bearer(azure.getSystemToken(config.appId))
             .header("Accept", "application/json")
+            .header("Content-Type", "application/json")
             .header("Nav-Call-Id", getOrCreateCorrelationIdFromThreadLocal())
-            .header("Nav-Personident", fnr.toString())
+            .body(request)
             .responseString()
 
         return result.fold(
             { json ->
-                deserialize<HentKontaktinformasjonRepsonse>(json).toKontaktinformasjon()
+                val feil = jsonNode(json).get("feil").get(fnr.toString())?.toString()
+                if (feil == null) {
+                    val personFunnet = jsonNode(json).get("personer").get(fnr.toString()).toString()
```

Comment:
Garantert bare ett svar her?

Reply:
Kan være greit å sjekke om `funnetPerson` ikke er tom streng og for en bedre feilmelding hvis ting allikevel skal tryne om feks fnr ikke er riktig eller kan ikke det skje?